portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,23 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+
+
+    <div id="mySidenav" class="sidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+        <a href="index.html">Home</a>
+        <a href="aboutMe.html">About Me</a>
+        <a href="#">Projects</a>
+        <a href="#">Resume</a>
+        <a href="#">Contact</a>
+    </div>
+
+    <span style="font-size:30px;cursor:pointer" onclick="openNav()">&#9776;</span>  
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hey, hello!</h1><p>
+      <img src="images/Untitled_Artwork.png" width="700" height="525"/><p>
```

Comment:
A couple of things here:

- consider giving the file a more descriptive name
- I don't see the image being added to the repo

Reply:
You don't see the .png being added?